### ISSUE TITLE
 (electron) Sending uncompressed crash reports is deprecated and will be removed in a future version of Electron. Set { compress: true } to opt-in to the new behavior. Crash reports will be uploaded gzipped, which most crash reporting servers support.

### ISSUE THREAD

**Issue by LoicKonan** [microsoft/vscode#117876]
Issue Type: <b>Bug</b>

I git this bug or warning whenever I try to run my program like this: 
./main.cpp

It let me use the mouse to hit the terminal and hit the run button but it won't let me run it with the keyboard.

VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:48:04.245Z)
OS version: Windows_NT x64 10.0.21322

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10700F CPU @ 2.90GHz (16 x 2904)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.91GB (10.24GB free)|
|Process Argv|--crash-reporter-id 216be74a-935e-4152-a347-e8cab5def58d|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (41)</summary>

Extension|Author (truncated)|Version
---|---|---
code-gnu-global|aus|0.2.2
emojisense|bie|0.8.0
markdown-checkbox|bie|0.1.3
markdown-preview-github-styles|bie|0.1.6
bracket-pair-colorizer-2|Coe|0.2.0
doxdocgen|csc|1.1.0
vscode-markdownlint|Dav|0.39.0
prettier-vscode|esb|5.9.2
vscode-pull-request-github|Git|0.23.1
banner-comments|hey|0.3.0
cppcomment|Hiy|1.0.3
comment-labels|jam|0.1.0
better-cpp-syntax|jef|1.15.5
chat|kar|0.35.0
vscode-gist|ken|3.0.3
vsc-python-indent|Kev|1.12.0
vsls-whiteboard|los|0.0.12
banner-comments-plus|lun|0.3.1
markdown-shortcuts|mdi|0.12.0
vscode-docker|ms-|1.10.0
python|ms-|2021.2.582707922
jupyter|ms-|2021.2.603412351
remote-containers|ms-|0.158.0
remote-ssh|ms-|0.64.0
remote-ssh-edit|ms-|0.64.0
remote-wsl|ms-|0.53.4
cmake-tools|ms-|1.6.0
cpptools|ms-|1.2.2
cpptools-extension-pack|ms-|1.0.0
powershell|ms-|2021.2.2
vsliveshare|ms-|1.0.3784
vsliveshare-audio|ms-|0.1.91
vsliveshare-pack|ms-|0.4.0
autodocstring|njp|0.5.4
LiveServer|rit|5.6.1
markdown-preview-enhanced|shd|0.5.16
pdf|tom|1.1.0
cmake|twx|0.0.17
gistfs|vsl|0.2.2
markdown-pdf|yza|1.4.4
markdown-all-in-one|yzh|3.4.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspyt653:30253241
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30256636
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
dockersubsetcf:30264245
pythontb:30260843

```

</details>

**Comment by IllusionMH** [comment]
/duplicate of #115888

See https://github.com/microsoft/vscode/issues/115888#issuecomment-780762061 for workaround

**Comment by deepak1556** [comment]
This will be fixed in the next release `1.54`
